src/estop.cpp

```diff
@@ -1,17 +1,17 @@
 #include "ros/ros.h"
 #include "std_srvs/Trigger.h"
-#include "JetsonXavierGPIO/jetsonGPIO.h"
+#include "JetsonXavierGPIO/jetsonGPIO.c"
```

Comment:
This is not what we want, this is very hacky, and I would not consider it correct. 

What we need to do is that in the JetsonXavierGPIU/jetsonGPIO.h we need to add define to allow for include in C++ code. 

Here's an explanation for it  : https://isocpp.org/wiki/faq/mixing-c-and-cpp#include-c-hdrs-personal

Reply:
Also, ideally, we should add abstraction to be able to have more portable code. So we could have a kind of BSP where we have `markhor_estop.h`, which would compile/include different type of header file depending on which robot we are building for.  But this might be out of scope for this PR

Reply:
Hum the document seems to say it's fine, so if you tell me that it works alright, I won't ask you to change it. 

http://wiki.ros.org/roscpp/Overview/Initialization%20and%20Shutdown

Reply:
It was tested, with .h it didn't work, so I copied the include from the old module markhor_estop, which was .c and worked. I  can still change it if needed.

Reply:
Yes, if you could change it that would be better thank you.